### PR TITLE
Moved Odyssey Data into Source

### PR DESCRIPTION
**Pull request by kaitlyndlee**
## Description

Moved serial number files into src code so that it is versioned. Updated Pipeline's unit test since it had hardcoded $ISIS3DATA references.

## Related Issue

#3727 

## Motivation and Context

This change is a part of moving the isis3data area, some files going into the source code.

## How Has This Been Tested?

Tested the apps and objects in Odyssey's directory. thmbasemap1's tests are failing because of a cube not being put into the output. thmproc's ir and vis are failing because of missing things in base.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Review by paarongiroux — APPROVED** [review]

**Comment by krlberry** [comment]
@kaitlyndlee What were the missing things in base causing the thmproc tests to fail?

**Review by krlberry — APPROVED** [review]

**Comment by kaitlyndlee** [comment]
This is the output from one of the tests:
```
**ERROR** Running Isis program [spiceinit] failed with return status [4].
**I/O ERROR** Data directory for mission [base] is not installed at your site.
Continuing ......
**ERROR** Running Isis program [cam2map] failed with return status [4].
**I/O ERROR** Unable to open [$ISIS3DATA/base/templates/maps/sinusoidal.map].
Continuing ......
**ERROR** Running Isis program [spiceinit] failed with return status [4].
**I/O ERROR** Data directory for mission [base] is not installed at your site.
Continuing ......
**ERROR** Running Isis program [spiceinit] failed with return status [4].
**I/O ERROR** Data directory for mission [base] is not installed at your site.
Continuing ......
**ERROR** Running Isis program [cam2map] failed with return status [4].
**ERROR** Running Isis program [spiceinit] failed with return status [4].
**I/O ERROR** Data directory for mission [base] is not installed at your site.
Continuing ......
**ERROR** Running Isis program [cam2map] failed with return status [4].
**I/O ERROR** Unable to open [$ISIS3DATA/base/templates/maps/sinusoidal.map].
Continuing ......
**ERROR** Running Isis program [spiceinit] failed with return status [4].
**I/O ERROR** Data directory for mission [base] is not installed at your site.
Continuing ......
**ERROR** Running Isis program [spiceinit] failed with return status [4].
**I/O ERROR** Data directory for mission [base] is not installed at your site.
Continuing ......
**ERROR** Running Isis program [cam2map] failed with return status [4].
**I/O ERROR** Unable to open [$ISIS3DATA/base/templates/maps/sinusoidal.map].
Continuing ......
**ERROR** Running Isis program [cam2map] failed with return status [4].
**I/O ERROR** Unable to open [./V01001002EDR.lev2.even.cub].
Continuing ......
**ERROR** Running Isis program [automos] failed with return status [4].
**I/O ERROR** Unable to open [./V01001002EDR.lev2.even.cub].
Continuing ......
**ERROR** Running Isis program [spiceinit] failed with return status [4].
**I/O ERROR** Data directory for mission [base] is not installed at your site.
Continuing ......
**ERROR** Running Isis program [spiceinit] failed with return status [4].
**I/O ERROR** Data directory for mission [base] is not installed at your site.
Continuing ......
**ERROR** PVL Keyword [Class] does not exist in [Group = Error].
Continuing ......
**ERROR** PVL Keyword [Class] does not exist in [Group = Error].
Continuing ......
**ERROR** Running Isis program [thmvistrim] failed with return status [4].
**I/O ERROR** Unable to open [./V05642009EDR.flat.even.cub].
Continuing ......
**ERROR** Running Isis program [thmvistrim] failed with return status [4].
**I/O ERROR** Unable to open [./V05642009EDR.flat.odd.cub].
Continuing ......
**ERROR** Running Isis program [spiceinit] failed with return status [4].
**I/O ERROR** Data directory for mission [base] is not installed at your site.
Continuing ......
**ERROR** Running Isis program [spiceinit] failed with return status [4].
**I/O ERROR** Data directory for mission [base] is not installed at your site.
Continuing ......
**ERROR** PVL Keyword [Class] does not exist in [Group = Error].
Continuing ......
**ERROR** PVL Keyword [Class] does not exist in [Group = Error].
Continuing ......
**ERROR** Running Isis program [cam2map] failed with return status [4].
**I/O ERROR** Unable to open [$ISIS3DATA/base/templates/maps/sinusoidal.map].
Continuing ......
**ERROR** Running Isis program [cam2map] failed with return status [4].
**I/O ERROR** Unable to open [./V05642009RDR.lev2.even.cub].
Continuing ......
**ERROR** Running Isis program [automos] failed with return status [4].
**I/O ERROR** Unable to open [./V05642009RDR.lev2.even.cub].
Continuing ......
```

File: isis/src/base/objs/Pipeline/Pipeline.truth [hunk 1]
```diff
@@ -1,94 +1,94 @@
 Simple Pipe
 PIPELINE -------> unitTest2 <------- PIPELINE
-thm2isis FROM="$ISIS3DATA/odyssey/testData/I00831002RDR.even.cub" TO="/work1/out.cub"
+thm2isis FROM="$odyssey/testData/I00831002RDR.even.cub" TO="/work1/out.cub"
```
Comment (scsides) [inline isis/src/base/objs/Pipeline/Pipeline.truth:3]: How is this full path on the TO parameter working?

Comment (krlberry) [reply isis/src/base/objs/Pipeline/Pipeline.truth:3]: Huh, I also have this question now.

Comment (scsides) [reply isis/src/base/objs/Pipeline/Pipeline.truth:3]: This isn't a real run. That's why it works.

**Review by scsides — APPROVED** [review]

**Pull request merged**